index.pug

```diff
@@ -256,6 +250,26 @@ html(lang='en')
                     p
                       | Need to provide those information - who did when change which segment? We get you covered.
 
+ 
+    section.integration
+      .container
+        .row
+          .col-md-6
+            h2 Easy integration for your operating tools
+            p There is an integration option for all tools. Start to reduce manual processes and everday tasks with easy integration of locize for busy people. We are your cost-effective translation management platform and help move your software automation to a new level. <a href="/how-it-works.html" title="">Javascript-localization</a> made easy for apps built with React, Angular and Vue.
```

Comment:
not only javascript, and not only react, angular and vue....
maybe add a hint, that this is not only limited to javascript and/or the listed frameworks...